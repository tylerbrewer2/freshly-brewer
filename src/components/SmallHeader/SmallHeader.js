import React from 'react';
import styled from 'styled-components';

const SmallHeader = styled.h1`
  font-family: 'AvenirNextBold', 'Helvetica', 'sans-serif';
  letter-spacing: 1px;
  font-size: 12px;
`

export default SmallHeader;
