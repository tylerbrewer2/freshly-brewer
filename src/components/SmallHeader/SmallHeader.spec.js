import React from 'react';
import renderer from 'react-test-renderer';

import SmallHeader from './SmallHeader';

it('renders correctly', () => {
  const tree = renderer.create(
    <SmallHeader>
      Test header
    </SmallHeader>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
