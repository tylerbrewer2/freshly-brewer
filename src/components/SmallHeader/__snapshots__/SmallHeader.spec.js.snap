exports[`test renders correctly 1`] = `
<h1
  className="sc-bdVaJa hkNTgp">
  Test header
</h1>
`;
